eng/New-TestResources.ps1

```diff
@@ -71,6 +71,27 @@ function Log($Message) {
     Write-Host ('{0} - {1}' -f [DateTime]::Now.ToLongTimeString(), $Message)
 }
 
+function Retry([scriptblock] $Action, [int] $Attempts = 5) {
+    $attempt = 0
+    $sleep = 1

```

Comment:
Sorry: I changed this for testing. It was supposed to start at 5 seconds. I'll change it back but don't want to push a change yet since the "net - keyvault - tests" pipeline is currently running and would otherwise be canceled.